packages/rancher-logging/rancher-logging.patch

```diff
@@ -128,16 +128,13 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/values
 +    tag: v0.2.2
 +  fluentbit:
 +    repository: rancher/fluent-fluent-bit
-+    tag: 1.5.4
++    tag: 1.6.1
 +  fluentbit_debug:
 +    repository: rancher/fluent-fluent-bit
-+    tag: 1.5.4-debug
++    tag: 1.6.1-debug
 +  fluentd:
 +    repository: rancher/banzaicloud-fluentd
-+    tag: v1.11.2-alpine-2
-+  syslog_forwarder:
```

Comment:
need to add back syslog and we can merge

Reply:
@paynejacob recommended that we remove this when I asked, so I'll double check if there's a reason to remove this.